spec/requests/api/v1/merchants/merchant_request_spec.rb

```diff
@@ -240,4 +240,74 @@
     expect(merchant_response['data'][0]['attributes']['name']).to eq(merchant2.name)
     expect(merchant_response['data'][1]['attributes']['name']).to eq(merchant3.name)
   end
+
+  it 'can get revenue for one merchant' do
```

Comment:
i could probably clean these up to use before each in future iterations for readability